.github/workflows/post-merge-tests.yml

```diff
@@ -119,13 +121,57 @@ jobs:
         gcc-fortran gmock-devel gtest-devel python3 python3-cython \
         python3-devel python3-numpy python3-pandas python3-pint python3-pip \
         python3-pytest python3-ruamel-yaml python3-scipy python3-scons \
-        python3-wheel sundials-devel yaml-cpp-devel
+        python3-wheel sundials-devel yaml-cpp-devel hdf5-devel highfive-devel
     - name: Build Cantera
       run: |
         scons build -j2 debug=n --debug=time \
         extra_inc_dirs=/usr/include/eigen3 f90_interface=y \
         libdirname=/usr/lib64 python_package=full system_eigen=y system_fmt=y \
-        system_sundials=y system_yamlcpp=y system_blas_lapack=y
+        system_sundials=y system_yamlcpp=y system_blas_lapack=y hdf_support=y
```

Comment:
Also `system_highfive=y`?

Reply:
I included it, but it is currently not used - the version is too old and the installer uses the private installation as a fallback solution (there is a pending fix). I added a separate test at the end of the runner that should illustrate this.

Reply:
Right, I forgot about that discussion. I think a comment explaining the reason for not specifying `system_highfive=y` despite installing the corresponding package would be sufficient. I would rather not introduce new tests that just fail out of the box.

Reply:
Sounds good.